cumulus/docs/release.md

```diff
@@ -75,56 +75,61 @@ function of the appropriate pallets.
 - If no indices have changed, every modules line should look something like `[Identity] idx 25 (calls 15)`
 
-**Note**: Adding new functions to the runtime does not constitute a breaking change
-as long as the indexes did not change.
+**Note**: Adding new functions to the runtime does not constitute a breaking change as long as the indexes did not
+change.
 
-**Note**: Extrinsic function signatures changes (adding/removing & ordering arguments) are not caught by the job, so those changes should be reviewed "manually"
+**Note**: Extrinsic function signatures changes (adding/removing & ordering arguments) are not caught by the job, so
+those changes should be reviewed "manually"
 
 ### Benchmarks
 
-The Benchmarks can now be started from the CI. First find the CI pipeline from [here](https://gitlab.parity.io/parity/mirrors/cumulus/-/pipelines?page=1&scope=all&ref=release-parachains-v9220) and pick the latest.
-[Guide](https://github.com/paritytech/ci_cd/wiki/Benchmarks:-cumulus)
+The Benchmarks can now be started from the CI. First find the CI pipeline from
+[here](https://gitlab.parity.io/parity/mirrors/cumulus/-/pipelines?page=1&scope=all&ref=release-parachains-v9220) and
+pick the latest. [Guide](https://github.com/paritytech/ci_cd/wiki/Benchmarks:-cumulus)
 
 ### Integration Tests
 
 Until https://github.com/paritytech/ci_cd/issues/499 is done, tests will have to be run manually.
-1. Go to https://github.com/paritytech/parachains-integration-tests and check out the release branch.
-E.g. https://github.com/paritytech/parachains-integration-tests/tree/release-v9270-v0.9.27
-for `release-parachains-v0.9.270`
+1. Go to https://github.com/paritytech/parachains-integration-tests and check out the release branch. E.g.
+https://github.com/paritytech/parachains-integration-tests/tree/release-v9270-v0.9.27 for `release-parachains-v0.9.270`
 2. Clone `release-parachains-<version>` branch from Cumulus
 3. `cargo build --release`
 4. Copy `./target/polkadot-parachain` to `./bin`
-5. Clone `it/release-<version>-fast-sudo` from Polkadot
-In case the branch does not exists (it is a manual process): cherry pick paritytech/polkadot@791c8b8 and run
-`find . -type f -name "*.toml" -print0 | xargs -0 sed -i '' -e 's/polkadot-vX.X.X/polkadot-v<version>/g'`
+5. Clone `it/release-<version>-fast-sudo` from Polkadot In case the branch does not exists (it is a manual process):
+cherry pick paritytech/polkadot@791c8b8 and run `find . -type f -name "*.toml" -print0 | xargs -0 sed -i '' -e
+'s/polkadot-vX.X.X/polkadot-v<version>/g'`
 6. `cargo build --release --features fast-runtime`
 7. Copy `./target/polkadot` into `./bin` (in Cumulus)
-8. Run the tests:
-	- Asset Hub Polkadot: `yarn zombienet-test -c ./examples/statemint/config.toml -t ./examples/statemint`
-	- Asset Hub Kusama: `yarn zombienet-test -c ./examples/statemine/config.toml -t ./examples/statemine`
+8. Run the tests: - Asset Hub Polkadot: `yarn zombienet-test -c ./examples/statemint/config.toml -t
```

Comment:
I think this should be a double indent here.

Reply:
Fixed in 95e69ad8c5